java/group-layers/README.md

```diff
@@ -2,19 +2,25 @@
 
 Group a collection of layers together and toggle their visibility as a group.
 
-![Group layers app](group-layers.png)
+![Image of group layers](group-layers.png)
 
-## Use case
+## Use case  
 
 Group layers communicate to the user that layers are related and can be managed together.
 
 In a land development project, you might group layers according to the phase of development.
 
+## How to use the sample
+
+The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
```

Comment:
Perhaps it's worth specifying that the layers menu is available from the button in the top right? If you interact with the map it closes the menu so users could conceivably get lost here.

Reply:
if you think it is helpful then I think it should go in.

Reply:
I am hedging however about saying top **right** as in devices set to locales for languages written left to right (e.g. Israel) it may display on the left.